docs/src/PolyhedralGeometry/pg_polyhedra.md

```diff
@@ -0,0 +1,53 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra.md"]
+```
+
+# Polyhedra
+
+## Introduction
+
+A subset $P \subseteq \mathbb{R}^n$ is called a (metric) polyhedron if it can be written as the intersection of finitely many closed affine halfspaces in $\mathbb{R}^n$.
+That is, there exists a matrix $A$ and a vector $b$ such that
+$$P = P(A,b) = \{ x \in \mathbb{R}^n \mid Ax \leq b\}.$$
```

Comment:
Maybe its a bot late, but we should think about xA leq b as well - after all, matrices are supposed to be row matrices...